cmake_modules/FindHWLOC.cmake

```diff
@@ -72,13 +72,14 @@ if(HWLOC_FOUND)
 
   #===============================================================================
   # Import Target ================================================================
-  if(NOT TARGET HWLOC::HWLOC)
-    add_library(HWLOC::HWLOC INTERFACE IMPORTED)
-  endif(NOT TARGET HWLOC::HWLOC)
+  if(NOT TARGET hwloc)
```

Comment:
Why the  name change here, this is not canonical (although the all caps we had before was not either), should be `hwloc::hwloc`